Example/SitumWayfinding/WayfindingController.swift

```diff
@@ -31,6 +31,14 @@ class WayfindingController: UIViewController {
                 self.performSegue(withIdentifier: "unloadWayfinding", sender: self)
             }
         )
+        self.library?.addLocationRequestInterceptor { (locationRequest: SITLocationRequest) in
+            locationRequest.useGlobalLocation = true;
+            let options: SITOutdoorLocationOptions = SITOutdoorLocationOptions()
```

Comment:
This appears to override settings of the app.. Is that true?

Why is this behaviour forced that way programmatically instead of letting the developer?